web/views/projects.py

```diff
@@ -281,3 +281,14 @@ def unpublish_project(request, pk):
     project.is_published = False
     project.save()
     return redirect(reverse_lazy('project', kwargs={'pk': project.id}))
+
+
+def dsw_list_projects(request):
+    #if data steward or admin -> list all the public projects
+    # if(request.user.is_admin() | request.user.is_datasteward()):
+    #     objects = Project.objects.all().filter(is_published=True)
+    objects = Project.objects.filter(local_custodians=request.user) | Project.objects.filter(company_personnel=request.user)
```

Comment:
from the efficiency point of view, it's better to add explicitly `is_published=True` to both queries here and remove line 291, or to move the line 291 before 290